src/stores/models/SyncInfo.js

```diff
@@ -16,10 +16,10 @@ export default class SyncInfo {
     this.percent = syncInfo.syncPercent;
     this.blockNum = syncInfo.syncBlockNum;
     this.blockTime = Number(syncInfo.syncBlockTime);
-    this.balances = _.map(syncInfo.addressBalances, (addressBalance) => new AddressBalance(addressBalance));
-    this.peerNodeCount = syncInfo.peerNodeCount;
+    this.peerCount = Number(syncInfo);
```

Comment:
is this supposed to be `syncInfo.peerNodeCount`?

Reply:
god good catch 
many thankssss